src/tests/update-attributes.js

```diff
@@ -0,0 +1,75 @@
+import { SharedArray } from 'k6/data';
+import { createEntity } from '../api/create-entity.js';
+import { updateAttributes } from '../api/update-attributes.js'
+import { uuidv4 } from 'https://jslib.k6.io/k6-utils/1.4.0/index.js';
+import { randomItem } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+export const options = {
+    vus: 10,
+    iterations: 100000,
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        update_attributes_duration: ['avg<200', 'p(95)<400']
+    },
+    setupTimeout: '5m',
+};
+
+const entities = new SharedArray('template entity', function () {
+    return JSON.parse(open('../data/template_entity.json')).entities;
+});
+
+const fragmentAttributes = new SharedArray('fragment update attributes', function () {
```

Comment:
it is called "Attribute Fragment", not "Fragment Attribute" (in the latter case, fragment is read as a verb, this is definitely not the same thing)